Sprinkle of Yeezus/Settings.swift

```diff
@@ -11,52 +11,68 @@ import UIKit
 import UserNotifications
 
 class SettingsPage: UIViewController {
-    
 
-    @IBOutlet weak var InfoLabel: UILabel!
-    @IBOutlet weak var DescButton: UIButton!
-    @IBOutlet weak var DescLabel: UILabel!
-    @IBOutlet weak var Switch: UISwitch!
+    @IBOutlet private weak var InfoLabel: UILabel!
+    @IBOutlet private weak var DescButton: UIButton!
+    @IBOutlet private weak var DescLabel: UILabel!
+    @IBOutlet private weak var Switch: UISwitch!
+    @IBOutlet private weak var TimePicked: UIDatePicker!
     
-    var notificationsOn = Bool()
+    private var sprinkleTimePicked = Date()
+    private let notificationCenter = UNUserNotificationCenter.current()
     
-    @IBAction func NotificationSwitch(_ sender: UISwitch) {
-        if Switch.isOn == true{
-            UNUserNotificationCenter.current().requestAuthorization(options: [.alert, .sound], completionHandler: {didAllow, error in})
-            
-            DescLabel.isHidden = false
-            DescButton.isHidden = false
-            
-            notificationsOn = true
-        } else {
-            DescLabel.isHidden = true
-            DescButton.isHidden = true
-            notificationsOn = false
-        }
+    // MARK: - Lifecycle
+    override func viewDidLoad() {
+        sprinkleNotifications(sprinkleList)
     }
     
-    @IBOutlet weak var TimePicked: UIDatePicker!
-    var sprinkleTimePicked = Date()
-    @IBAction func TimePicker(_ sender: UIDatePicker){
-        sprinkleTimePicked = TimePicked.date
+    // MARK: - IBAction
+    @IBAction private func NotificationSwitch(_ sender: UISwitch) {
+        if sender.isOn {
```

Comment:
I noticed that the code between the `if` and `else` statements were relatively the same (just opposites of one another), with the exception of this piece where you `requestAuthorization` from the Notification Center. For that reason, I extracted the `.isHidden` code for hiding/unhiding the elements into a `updateUI()` function because it gets called in multiple places. I'll elaborate on that later.